services/table-foreign-keys-analyzer.js

```diff
@@ -49,15 +49,44 @@ function TableForeignKeysAnalyzer(databaseConnection, schema) {
         break;
       case 'mysql':
         query = `
-          SELECT
-            TABLE_NAME AS "tableName",
-            COLUMN_NAME AS "columnName",
-            CONSTRAINT_NAME AS "constraintName",
-            REFERENCED_TABLE_NAME AS "foreignTableName",
-            REFERENCED_COLUMN_NAME AS "foreignColumnName"
-          FROM INFORMATION_SCHEMA.KEY_COLUMN_USAGE
-          WHERE TABLE_SCHEMA = :databaseName
-            AND TABLE_NAME = :table;`;
+        SELECT constraintName,
+               tableName,
+               columnName,
+               columnType,
+               foreignTableName,
+               foreignColumnName,
+               CASE
+                 WHEN cast('[null]' AS json) = uniqueIndexes THEN NULL
+                 ELSE uniqueIndexes
+               END AS uniqueIndexes
+        FROM (
+          SELECT tableConstraints.constraint_name AS constraintName,
+                 tableConstraints.table_name AS tableName,
+                 keyColumnUsage.column_name AS columnName,
+                 tableConstraints.constraint_type AS columnType,
+                 keyColumnUsage.referenced_table_name AS foreignTableName,
+                 keyColumnUsage.referenced_column_name AS foreignColumnName,
+                 JSON_ARRAYAGG(uidx.uniqueIndexes) AS uniqueIndexes
+          FROM information_schema.table_constraints AS tableConstraints
+          JOIN information_schema.key_column_usage AS keyColumnUsage
+            ON tableConstraints.table_name = keyColumnUsage.table_name
+            AND tableConstraints.constraint_name = keyColumnUsage.constraint_name
+          LEFT OUTER JOIN (
+            SELECT distinct uidx.index_name AS indexName,
+                   uidx.table_name AS tableName,
+                   JSON_ARRAYAGG(uidx.column_name) AS uniqueIndexes
+            FROM information_schema.statistics AS uidx
+            WHERE index_schema = 'lumber-sequelize-test'
+              AND uidx.non_unique = 0
+              AND uidx.index_name != 'PRIMARY'
+            GROUP BY tableName, indexName) AS uidx
+            ON uidx.tableName = tableConstraints.table_name
+           WHERE tableConstraints.table_schema = 'lumber-sequelize-test'
+              AND tableConstraints.table_name = 'addresses'
```

Comment:
Not sure, but maybe replace this with: `:databaseName` and `:table`

Reply:
Fixed